Sources/SwifterSwift/UIKit/UIColorExtensions.swift

```diff
@@ -18,6 +18,64 @@ public extension UIColor {
         }
     }
     #endif
+
+    /// SwifterSwift: A computed property that returns the hexadecimal string representation of the color.
+    ///
+    /// This property converts the color to its RGB components and then formats them as a hex string
+    /// in the format `#RRGGBB`. The alpha component is ignored.
+    ///
+    /// - Returns: A string representing the color in hexadecimal format, prefixed with `#`.
+    ///
+    /// Example:
+    /// ```
+    /// let color = UIColor.red
+    /// print(color.hexString) // Output: "#ff0000"
+    /// ```
+    var hexString: String {
```

Comment:
Doesn't this already exists on the project?https://github.com/SwifterSwift/SwifterSwift/blob/3b2a1725e967498edadb84e6271bb85430c1961c/Sources/SwifterSwift/Shared/ColorExtensions.swift#L91

Reply:
Your are right. When I was checking on the project I only rechearched inside the UIColor extension, but his class that you mention is a ealias `public typealias SFColor = UIColor`. So in this project has 2 places to manage UIColor extension:
- Source/SwifterSwift/UIkit/UIColorExtensions
- Source/SwifterSwift/Shared/ColorExtensions

In this case, the var that I added is not necessary, I will remove.

Reply:
Also I will add tests cases